dataloadgen.go

```diff
@@ -75,28 +86,38 @@ type loaderBatch[KeyT comparable, ValueT any] struct {
 	errors        []error
 	fetchExecuted bool
 	done          chan struct{}
+	contexts      []context.Context
 }
 
 // Load a ValueT by key, batching and caching will be applied automatically
-func (l *Loader[KeyT, ValueT]) Load(key KeyT) (ValueT, error) {
-	return l.LoadThunk(key)()
+func (l *Loader[KeyT, ValueT]) Load(ctx context.Context, key KeyT) (ValueT, error) {
+	return l.LoadThunk(loadContext, key)()
```

Comment:
@vikstrous2 what's the best way to pass in a name from a call to `Load` outside?